### PR TITLE
make it easier to view and edit email templates

### LINKED ISSUE #139: Create dev page for testing email templates
So that we can more quickly develop email templates we should be able to see a version with dummy content.

I assume gmail uses some base css, so maybe just use normalize.css or rip the styles from gmail.

### PR DESCRIPTION
**Pull request by meandavejustice**
fixes #139 

Doesn't add any styles from the gmail compose window, it looks very similar already. Happy to add it if we find it necessary. ( it has normalize because bootstrap is included for the tabs)

### CODE REVIEW

**Comment by meandavejustice** [comment]
template-check-email-tabs

File: README.md [hunk 1]
```diff
@@ -35,6 +35,11 @@ If you are developing, note that the `.jsx` file will not trigger a reload on it
 npm run watch
 ```
 
+If you want to view the template in browser (easier for style tweaks):
+```sh
```
Comment (johngruen) [inline README.md]: @meandavejustice sh?

**Comment by meandavejustice** [comment]
This could use some styling, but this is the minimum needed to start onto reviewing the templates. Which seems to be a much higher priority, I can swing back around and clean it up further at a later date.

**Comment by johngruen** [comment]
WFM, let's go ahead and getting it merged. I can tweak styles after. Just fix that `sh`

File: README.md (hunk 1 above)
Comment (meandavejustice) [reply README.md]: shell

Comment (meandavejustice) [reply README.md]: shell

**Pull request merged**